scalableminds/webknossos
# Enable mappings in volume/hybrid tracings

... that are based on an existing segmentation. For volume tracings with a completely new segmentation, it doesn't really make sense to apply mappings from the server.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a new volume tracing based on the existing segmentation! on a dataset that has mappings.
  - You should be able to use the mappings UI to list the available mappings and select specific mappings.
  - You should be able to programatically add a new mapping, e.g.: using `webknossos.apiReady(3).then(async (api) => {api.data.setMapping(api.data.getVolumeTracingLayerName(), {7353: 1337})})` (replace first id with one you can see in the view)
- Selecting mappings in a skeleton tracing or when viewing a dataset should still work

### Issues:
- fixes #3863 

------
